metrics-scm-1.rockspec

```diff
@@ -22,29 +22,31 @@ build = {
-        ['metrics.collectors.histogram']                 = 'metrics/collectors/histogram.lua',
-        ['metrics.plugins.graphite']                     = 'metrics/plugins/graphite/init.lua',
-        ['metrics.plugins.prometheus']                   = 'metrics/plugins/prometheus/init.lua',
-        ['metrics.plugins.json']                         = 'metrics/plugins/json/init.lua',
-        ['metrics.default_metrics.tarantool']            = 'metrics/default_metrics/tarantool/init.lua',
-        ['metrics.default_metrics.tarantool.fibers']     = 'metrics/default_metrics/tarantool/fibers.lua',
-        ['metrics.default_metrics.tarantool.info']       = 'metrics/default_metrics/tarantool/info.lua',
-        ['metrics.default_metrics.tarantool.memory']     = 'metrics/default_metrics/tarantool/memory.lua',
-        ['metrics.default_metrics.tarantool.network']    = 'metrics/default_metrics/tarantool/network.lua',
-        ['metrics.default_metrics.tarantool.operations'] = 'metrics/default_metrics/tarantool/operations.lua',
-        ['metrics.default_metrics.tarantool.replicas']   = 'metrics/default_metrics/tarantool/replicas.lua',
-        ['metrics.default_metrics.tarantool.runtime']    = 'metrics/default_metrics/tarantool/runtime.lua',
-        ['metrics.default_metrics.tarantool.slab']       = 'metrics/default_metrics/tarantool/slab.lua',
-        ['metrics.default_metrics.tarantool.spaces']     = 'metrics/default_metrics/tarantool/spaces.lua',
-        ['metrics.default_metrics.tarantool.system']     = 'metrics/default_metrics/tarantool/system.lua',
-        ['metrics.default_metrics.tarantool.utils']      = 'metrics/default_metrics/tarantool/utils.lua',
+        ['metrics']                                         = 'metrics/init.lua',
+        ['metrics.registry']                                = 'metrics/registry.lua',
+        ['metrics.http_middleware']                         = 'metrics/http_middleware.lua',
+        ['metrics.collectors.shared']                       = 'metrics/collectors/shared.lua',
+        ['metrics.collectors.average']                      = 'metrics/collectors/average.lua',
+        ['metrics.collectors.counter']                      = 'metrics/collectors/counter.lua',
+        ['metrics.collectors.gauge']                        = 'metrics/collectors/gauge.lua',
+        ['metrics.collectors.histogram']                    = 'metrics/collectors/histogram.lua',
+        ['metrics.plugins.graphite']                        = 'metrics/plugins/graphite/init.lua',
+        ['metrics.plugins.prometheus']                      = 'metrics/plugins/prometheus/init.lua',
+        ['metrics.plugins.json']                            = 'metrics/plugins/json/init.lua',
+        ['metrics.default_metrics.tarantool']               = 'metrics/default_metrics/tarantool/init.lua',
+        ['metrics.default_metrics.tarantool.cpu']           = 'metrics/default_metrics/tarantool/cpu.lua',
```

Comment:
cpu metrics is just a beginning. sooner or later there will be memory metrics, network stats etc. I propose to move cpu.lua to a different package, let it be `metrics/psutil/cpu.lua`

Reply:
I think it makes sense to move `metrics/default_metrics/tarantool/utils.lua` in a different package too, because it may be used in other packages.

Reply:
good point but let `utils.lua` refactoring be a subject of different pr 
will make an issue